pkg/tfshim/sdk-v2/cty_test.go

```diff
@@ -415,20 +417,20 @@ func TestMakeResourceRawConfig(t *testing.T) {
 			},
 			expected: cty.ObjectVal(map[string]cty.Value{
 				"allowed_logout_urls": cty.NullVal(cty.List(cty.String)),
-				"change_password": cty.ListValEmpty(cty.Object(map[string]cty.Type{
+				"change_password": cty.NullVal(cty.List(cty.Object(map[string]cty.Type{
```

Comment:
The fix to cty.go replaced ListValEmpty with NullVal in some of the inputs to the SimpleDiff method, which sounds benign but I wasn't 100% sure so just in case I subjected the fix to flagging also.